perf/benchmarkapps/README.md

```diff
@@ -11,4 +11,20 @@ Benchmarks can be run locally from the command line.
 Example of running client using Grpc.Net.Client against Grpc.AspNetCore server:
 
 1. **Launch server:** dotnet run -c Release -p .\perf\benchmarkapps\GrpcAspNetCoreServer\ --protocol h2c
-2. **Launch client:** dotnet run -c Release -p .\perf\benchmarkapps\GrpcClient\ -- -u http://localhost:5000 -c 10 --streams 50 -s unary -p h2c --grpcClientType grpcnetclient
\ No newline at end of file
+2. **Launch client:** dotnet run -c Release -p .\perf\benchmarkapps\GrpcClient\ -- -u http://localhost:5000 -c 10 --streams 50 -s unary -p h2c --grpcClientType grpcnetclient
+
+## QpsWorker
+
+The `QpsWorker` runs in the [gRPC benchmark environment](https://grpc.io/docs/guides/benchmarking/). The worker hosts gRPC services which are used to start a benchmark server or client.
+
+* `driver_port` - Port for the worker. Consistent with other drivers.
+* `LogLevel` - Logging level of the client and server runners. Optional. Defaults to no logging.
+
+```cmd
+dotnet run -c Release -- --LogLevel Debug --driver_port 5000
+```
+
+To test a running worker:
+
+* `worker-start-server.ps1` starts a server.
```

Comment:
while this might be useful for experimenting, note that the official way of running the benchmarks is to use the `qps_json_driver`. As we've seen there are subtleties in the behavior that will only become apparent when the qps driver is used.
We might wanna add a note that the official benchmarks use the qps_json_driver, but since it's written in C++ (and AFAIK is only buildable on unix) it might be bit tricky to use for quick experimenting.